### DISCUSSION TITLE
help needed using ILI9488 with esp32-wroom-32d

### DISCUSSION THREAD

**Discussion by saunderschris** [Bodmer/TFT_eSPI#985] [Q&A - General]
Hi,
I am trying to configure the ILI9488 with the esp32 module using your library. I have changed the User_setup.h file to define the pins and defined the driver to ILI9488. I have commented out the other drivers and default pins on the default configuration. I have added a serial print script at the beginning of the example script to confirm the pin selection is correct and has been imported into my ardiuno script. However,
[user_setup_h.docx](https://github.com/Bodmer/TFT_eSPI/files/5933505/user_setup_h.docx)
 I can get nothing to display (except backlight) on the screen (480x320 3.5" SPI TFT Module). I have not connected the touch or MISO pins at this stage. I am a relative beginner and was looking for some advice if you have time.
Many thanks, chris

**Comment by saunderschris** [comment]
Have got the tft working and I have managed to get the touchscreen calibration working. Not sure how to close this thread!

**Reply by manuferro** [reply]
Hi,
  can you describe what solve the problem? I'm having thr same issues.

Thanks
